_posts/2020-01-27-why-upgrade-your-rails-application.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Why Is It Important to Upgrade Your Rails Application?"
+date: 2020-01-31 13:00:00
+categories: ["rails", "upgrades"]
+author: abizzinotto
+---
+
+Technology evolves quickly. When you first created your Rails application, it's likely you did it using the latest version of Rails. As time passes, however, new Rails versions come out and your application becomes more and more out of touch with the latest version. But why is this important? Why does it matter?
+
+There are several different reasons to [upgrade your Rails application](https://www.fastruby.io/blog/rails/upgrades/southeast-ruby-upgrade-rails-workshop.html). In this article, I’ll list what we consider to be the most important ones.
+
+<!--more-->
+
+## Security
+
+Security is a big issue. You want your application to be as safe as possible and, to accomplish that, one of the pivotal things to do is to make sure Ruby on Rails is up to date. Ruby on Rails is the application framework underlying everything your web application does. 
+
+As new threats arise, new versions contain security patches to protect you against them. Being up to date on your Rails version allows you to rapidly apply the newest security patches to your application.
```

Comment:
@ABizzinotto Maybe you can add a link to this service? https://dependabot.com 

It's quite useful because it will send you minor version bump pull requests automatically.

Reply:
Another thing that you could mention is that, if you are running Rails 4.2, then you are probably exposed to these vulnerabilities: https://www.fastruby.io/rails-4-2-vulnerabilities

Reply:
Done